web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -75,6 +76,17 @@
             :rules="awardNumberRules"
           />
         </div>
+        <div v-if="embargoed===false">
+          <v-combobox
```

Comment:
What are your thoughts on using a `v-select` instead of `v-combobox` here? IMO the extra autocomplete features `v-combobox` provides just confuse things, since we there's only two license types to choose from. Also, fwiw the meditor uses a `v-select` for licenses.

Reply:
sorry, I missed the comments.... I'm not sure if I had any specific thoughts, will try to use v-select